src/main/resources/static/js/subscription.js

```diff
@@ -861,9 +861,15 @@ jQuery(document).ready(function () {
             $('#subscriptionNameInput').addClass("is-invalid");
             error = true;
         }
-
+        
+        // When this regExp need to be changed then remember to change the one in the
+        // back-end (invalidSubscriptionNameRegex in SubscriptionValidator.java), which do the same
+        // invalid subscription name check. The two
+        // regEx always need to be the same for ensuring the same check.
         // /(\W)/ Is a regex that matches anything that is not [A-Z,a-z,0-8] and _.
```

Comment:
Shouldn't it be possible to use dash "-" in SubscriptionName also, or is it only underscore?

Reply:
This regEx doesnt accept dash (-) in the subscription name... as we discussed, it shouldnt be in the name.

Reply:
Ok,,, I just trying to think of what and which subscription name variants that user want to use.